filebeat/harvester/config.go

```diff
@@ -45,6 +46,7 @@ type harvesterConfig struct {
 	CloseRemoved         bool                       `config:"close_removed"`
 	CloseRenamed         bool                       `config:"close_renamed"`
 	CloseEOF             bool                       `config:"close_eof"`
+	CloseTimeout         time.Duration              `config:"close_timeout" validate:"min=0,nonzero"`
```

Comment:
It's not. using `nonzero` in `validate` requires `close_timeout > 0` in config files.